scripts/README.md

```diff
@@ -0,0 +1,13 @@
+# Scripts
+
+- `create_service.sh`
+  - creates managed instance group and backend service
+- `create_health_check.sh`
+  - creates health check
```

Comment:
Maybe add some more description here?  You're just restating the name of the script.  :)  A little more detail about what a health check is and/or when you might need to use it would be good.

Reply:
Done